dist/common/cassandra.in.sh

```
@@ -7,6 +7,7 @@ function cp_conf_dir {
     cp -a "$1"/*.xml "$2"  2>/dev/null || true
     cp -a "$1"/*.options "$2"  2>/dev/null || true
     cp -a "$1"/*.properties "$2"  2>/dev/null || true
+    cp -a "$1"/cassandra-env.sh "$2"  2>/dev/null || true
```

If it's mandatory why mask the error?

---

@tarzanek

---

hmm, that is true, it could be without error masking, though copying .yaml is "masked" too, can we open a bug for @amnonh to revisit this? I just continued with extending his patch ( https://github.com/scylladb/scylla-tools-java/pull/108 ), we can revisit his patch